install/tools/man/ipa-pkinit-manage.1

```diff
@@ -0,0 +1,34 @@
+.\"
+.\" Copyright (C) 2017  FreeIPA Contributors see COPYING for license
+.\"
+.TH "ipa-pkinit-manage" "1" "Jun 05 2017" "FreeIPA" "FreeIPA Manual Pages"
+.SH "NAME"
+ipa\-pkinit\-manage \- Enables or disables PKINIT
```

Comment:
I would add here a comment that this enables PKINIT on single IPA master only, not in the whole topology. Otherwise users may get confused.